### ISSUE TITLE
Early Intentional draws and potential Pairing Algorithm changes

### ISSUE THREAD

**Issue by fbatista** [Juizes-MTG-Portugal/Juizes-MTG-Portugal.github.io#19]
Given the fact that wins are extremely difficult to achieve in 4-FFA, it is common for players in short to mid-size events to start intentionally drawing very early in the event, in a way to secure a position in the top cut.

This Solution attempts to create even pods in round 2, using the snake pairing algorithm, which has a chance to counteract the random pairings from round 1. This also increase the chance to get more players with wins. The "downside" is that if 2 really good players won round 1, it's likely they both win round 2 (although the odds are not in their favor, since again, winning games is difficult), but if they that happens, we are happy with the outcome as the really good players will get a great advantage for the remaining of the tournament.

---

### As such a solution is being discussed, presented below:

If the tournament has 3 or more rounds in the "swiss portion" of the event, apply the following model:

- Round 1, random pairing.
- Round 2, snake pairing, the same as used in the Single Elimination portion.
- Round 3+ "Swiss" pairing.

- Top cut for Single Elimination.

#### For the "Swiss" pairing:

1. Pairing by Standing: [1,2,3,4] [5,6,7,8] [9,10,11,12] ... [Nth-2, Nth-1, Nth] (group of 3 at the end),
  1.1. but try to avoid rematching, meaning there's a possibility of swapping players between groups, doing pair-up / pair down.
2. Then, ensure that in the last groups, the players who have already played in the group of 3 can be swapped with others who haven't played yet,
  2.1. but ensuring that there's no rematching in this situation. (in other words, repeat the process similar to the one used in point 1.1., but executed in reverse mode, starting from the last group).
3. Scoring: win = 7; loss = 0; draw = 1

#### Tiebreaker:

1. Opponent's Match Win %, but each player contributes at least (points for draw / points for win) to the OPP Match Win % of the opponents. (14.29% for 7 points per win)
2. In a group of 3, a 4th player is considered who will ALWAYS have (points for draw / points for win) win% for the purposes of calculating the OPPONENT's Match Win %.
